packages/reporting-e2e/docker/reporting-server-migrations/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG BASE_IMAGE=docker.pkg.github.com/open-rmf/rmf-web/reporting-server
+FROM $BASE_IMAGE
+
+SHELL ["bash", "-c"]
+
+COPY packages/reporting-server/migrations/ /root/migrations/
```

Comment:
This shouldn't be needed as `reporting-server` should install whatever is necessary.

Reply:
Ok! So I prepacked the project first and then I copied the `.whl` to the dockerfile, from there I installed the requirements. 7352ab1db7752bf0d53a55daccb51cc98e7acc0c

I just don't want to install Nodejs and Lerna for this pure python-aerich related test. I think I'll take more time and it'll use more resources. But if that is the correct way of doing it, I will. 

Let me know what you think

Reply:
referencing this

> I guess the primary way to run this is in a docker container, so it is not designed to be usable with the "old school pip3 install"?

I guess if there is no plans to make this work with `pip3 install`. Then the whole process of building a wheel is unneeded complexity, we can just copy the source into the image.

But on the other hand, if we still want to support `pip3 install`, then we should not be doing any extra copying of sources into the image. We should just run `pip3 install` and everything needed should be installed, i.e. the wheel should contain the migration scripts.